demos/MoAE/options_task-auditory.json

```diff
@@ -0,0 +1,50 @@
+{
+    "STC_referenceSlice": [],
+    "anatReference": {
+        "type": "T1w",
+        "session": 1
```

Comment:
Oh this defines all the options to run your batch in case you don't want to have a `getOption` function.

More details about this should be here but currently are not (need to add more info) : `loadAndCheckOptions`
https://cpp-bids-spm.readthedocs.io/en/latest/function_description.html#src.utils.loadAndCheckOptions

In the JOBS folder you only have the mat files of the matlabbatch and json files that capture more information about your "computing environments" : OS version, matlab version...